dashboard/app/models/script.rb

```diff
@@ -1141,6 +1140,11 @@ def update_text(script_params, script_text, metadata_i18n, general_params)
     end
   end
 
+  def write_script_json
+    script_json_filepath = "config/scripts_json/#{name}.script_json"
```

Comment:
Sorry for suggesting this now but could we use `Rails.root` here? It makes this callable from outside the Rails directory (i.e. the import script)

```suggestion
    script_json_filepath = "#{Rails.root}/config/scripts_json/#{name}.script_json"
```

Reply:
I tried this suggestion and it broke a bunch of tests (ScriptTest and ScriptsControllerTest, IIRC). I've reverted it for now. @bethanyaconnor may I leave you to figure out how to make this change in your PR? I just force-pushed the branch of this PR, apologies for that. I shouldn't have to force-push again, so it should be safe to rebase onto this branch again now.

Reply:
FWIW I verified that the force-push didn't change any code, it just cleaned up a fix from a bad merge.

Reply:
Sounds good!